content/correspondence/reference/security/_index.nb.md

```
@@ -8,8 +8,23 @@ weight: 50
 ---
 
-<!--
-{{<children />}}
--->
+## Etterlevelse av beste praksis for informasjonssikkerhet i Altinn
 
-TBD
\ No newline at end of file
+Altinn Correspondence følger beste praksis for informasjonssikkerhet
+slik dette er redegjort for under 
+[Altinn sikkerhet ](/nb/security/).
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)